internal/beater/beater.go

```diff
@@ -700,6 +700,7 @@ func (s *Runner) newFinalBatchProcessor(
 	}
 	esConfig.FlushInterval = time.Second
 	esConfig.Config = elasticsearch.DefaultConfig()
+	esConfig.MaxIdleConnsPerHost = 10
```

Comment:
Why setting it to `10` here and then only later to `esConfig.MaxRequests`?

Reply:
Other default values are set in the `elasticsearch/config.go`; IMO the default value should also be set there.

Reply:
So in this case, shouldn't this default value then always be set via `esConfig.MaxRequests` in line `724`? Why setting it to a hardcoded default, that easily gets outdated (as the docappender default value comes from a different source)?